python/lsst/pipe/tasks/functors.py

```diff
@@ -1620,66 +1601,13 @@ def _func(self, df):
                                                df[self.photoCalibCol], df[self.photoCalibErrCol])
 
 
-class LocalMagnitude(LocalPhotometry):
-    """Compute calibrated AB magnitudes using the local calibration value.
-
-    See also
-    --------
-    LocalNanojansky
-    LocalNanojanskyErr
-    LocalMagnitude
-    LocalMagnitudeErr
-    """
-
-    @property
-    def columns(self):
-        return [self.instFluxCol, self.photoCalibCol]
-
-    @property
-    def name(self):
-        return f'mag_{self.instFluxCol}'
-
-    def _func(self, df):
-        return self.instFluxToMagnitude(df[self.instFluxCol],
-                                        df[self.photoCalibCol])
-
-
-class LocalMagnitudeErr(LocalPhotometry):
-    """Compute calibrated AB magnitude errors using the local calibration value.
-
-    See also
-    --------
-    LocalNanojansky
-    LocalNanojanskyErr
-    LocalMagnitude
-    LocalMagnitudeErr
-    """
-
-    @property
-    def columns(self):
-        return [self.instFluxCol, self.instFluxErrCol,
-                self.photoCalibCol, self.photoCalibErrCol]
-
-    @property
-    def name(self):
-        return f'magErr_{self.instFluxCol}'
-
-    def _func(self, df):
-        return self.instFluxErrToMagnitudeErr(df[self.instFluxCol],
-                                              df[self.instFluxErrCol],
-                                              df[self.photoCalibCol],
-                                              df[self.photoCalibErrCol])
-
-
 class LocalDipoleMeanFlux(LocalPhotometry):
     """Compute absolute mean of dipole fluxes.
 
     See also
     --------
     LocalNanojansky
     LocalNanojanskyErr
-    LocalMagnitude
-    LocalMagnitudeErr
     LocalDipoleMeanFlux
```

Comment:
Can prolly also delete this "see itself"!

Reply:
Ah, I missed some of the self references. Thanks for catching them Lauren!